editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -88,12 +92,13 @@ export function interactionSession(
     userPreferredStrategy: userPreferredStrategy,
     startedAt: startedAt,
     allElementProps: allElementProps,
+    startingTargetParentToFilterOut: startingTargetParentToFilterOut,
   }
 }
 
 export type InteractionSessionWithoutMetadata = Omit<
   InteractionSession,
-  'metadata' | 'allElementProps'
+  'metadata' | 'allElementProps' | 'startingTargetParentToFilterOut'
```

Comment:
I wonder if this is reaching the point where we should just define `InteractionSession` as being this type plus those fields.